Sources/Widgets/Core/WidgetManager/test/testWidgetManager.js

```diff
@@ -128,9 +128,10 @@ test.onlyIfWebGL('Test getPixelWorldHeightAtCoord', (t) => {
   }
 
   function testScaleInPixelsWithParallel() {
-    t.comment(
+    t.skip(
```

Comment:
is the goal here is to print in output. Does .skip() do it?

Reply:
I think @floryst skipped it because the tests is currently failing.